ask-sdk-core/README.rst

```diff
@@ -40,13 +40,13 @@ Usage and Getting Started
 -------------------------
 
 Getting started guides, SDK Features, API references, samples etc. can
-be found at ``Read The Docs`` <https://alexa-skills-kit-python-sdk.readthedocs.io/en/latest/>
+be found at `Read The Docs <https://alexa-skills-kit-python-sdk.readthedocs.io/en/latest/>`
 
 
 Got Feedback?
 -------------
 
 - We would like to hear about your bugs, feature requests, questions or quick feedback.
-  Please search for the ``existing issues`` <https://github.com/alexa/alexa-skills-kit-sdk-for-python/issues> before opening a new one. It would also be helpful
-  if you follow the templates for issue and pull request creation. Please follow the ``contributing guidelines`` <https://github.com/alexa/alexa-skills-kit-sdk-for-python/blob/master/CONTRIBUTING.md>!
-- Request and vote for ``Alexa features`` <https://alexa.uservoice.com/forums/906892-alexa-skills-developer-voice-and-vote>!
+  Please search for the `existing issues <https://github.com/alexa/alexa-skills-kit-sdk-for-python/issues>` before opening a new one. It would also be helpful
```

Comment:
The URL format to represent hyperlinks is  `<NAME> <URL LINK>`_.
So please add _(underscore) at the end.
You can refer this for exaple: https://raw.githubusercontent.com/alexa/alexa-skills-kit-sdk-for-python/master/ask-sdk-jinja-renderer/README.rst

for eg `existing issues <https://github.com/alexa/alexa-skills-kit-sdk-for-python/issues>`_

Reply:
Done! Thanks for catching this.